torchaudio_contrib/functional.py

```diff
@@ -287,6 +288,7 @@ def amplitude_to_db(x, ref=1.0, amin=1e-7):
     Returns:
         (Tensor): same size of x, after conversion
     """
+    x = x.pow(2.)
```

Comment:
@keunwoochoi So I noticed that our implementation of `amplitude_to_db` was actually `power_to_db`. I added the appropiate `** 2` here and `sqrt()` for `db_to_amplitude`, as well as for their tests.